app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ProfilesController < ApplicationController
+  def show
+    @user = User.find(params[:id])
```

Comment:
I know we talked about a `Profile` PORO here that looked up users by a `username` instead of DB ID (to prevent session hijacking). Are there future plans for that?

Reply:
Yes. I got derailed reading about options in the docs and forgot about the PORO piece of the Profile discussion.